app/posts/find-teacher-training/2025-11-05-improving-filtering-and-sorting.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Improving how candidates can filter and sort courses
+description: Researching filter components across government and designing and testing a solution to help candidates filter courses and see results.
+date: 2025-11-05
+tags:
+  - find
+  - filters
+  - filterable find
+---
+
+> **Relates to:**
+> [Filterable Find postgraduate training courses (Find) concept](/find-teacher-training/filterable-find-concept/) - October 2022
+
+## Context
+
+Research in January 2025 on [improving the ability to browse and search for courses](https://becoming-a-teacher.design-history.education.gov.uk/find-teacher-training/improving-the-ability-to-browse-and-search-for-courses-round-1-user-research/) revealed some usability issues with the filters on Find.
+
+### 1. Seeing the ability to filter on mobile
+
+In our research, Candidates on mobile phones did not notice ‘show filters’ option until it was pointed out.
+
+<img alt="Screenshot of the live site on a mobile with a grey button to open a panel that displays ways to filter courses." src="live-site-filters-button.png" style="width:50%;">
+
+In March 2025, the design ops team at DfE explored [applying filters consistently in DfE](https://dfedigital.blog.gov.uk/2025/03/03/applying-filters-consistently-in-dfe/), where they found several instances of users missing the “Show filters” button when using a mobile device.
+
+They thought users may miss this grey secondary action button for a few reasons, including:
+
+- it looks inactive
+- the colour of the button is too similar to the background colour of the filters
+- the button is too far away from the results it relates to
+
+There were also usability issues with navigating the filters on Mobile, such as the panel overlaying page content, and not being clear on how to close the filter panel once opened.
+
+<img alt="Screenshot of the live site on a mobile with the filter panel open displaying the ways to filter courses." src="live-site-filters-panel.png" style="width:50%;">
+
+### 2. Seeing what filters have been selected
+
+In our research, there was an expectation for applied filter labels to be present, so that they could see what they had selected.
+
+Previous design work was done in 2022 on [exploring selected filter tags](https://becoming-a-teacher.design-history.education.gov.uk/find-teacher-training/exploring-selected-filter-tags/).
```

Comment:
We use relative URLs for links within the design history site and absolute URLs for external links.

```suggestion
Previous design work was done in 2022 on [exploring selected filter tags](/find-teacher-training/exploring-selected-filter-tags/).
```